package.json

```diff
@@ -3,6 +3,17 @@
   "version": "1.2.0",
   "description": "Module Base template for create new modules for MagicMirror",
   "main": "MagicMirror-Module-Template.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gjonesme/MMM-WeatherAlerts.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "MagicMirror²",
+    "Weather",
+    "Alerts",
+    "OpenWeatherMap",
```

Comment:
Trailing comma is not allowed in JSON files. With that the file can not be parsed. Please remove the comma at the end of line 15 :slightly_smiling_face: